server/src/controllers/auth.js

```diff
@@ -88,8 +88,12 @@ module.exports = {
 					ticket: userTicket
 				}
 			}).then((sres) => {
-				if (sres.data.response.params.result === 'OK') {
-					if (idToVerify === sres.data.response.params.steamid) {
+				const params = sres.data.response.params;
+				if (!params)
```

Comment:
This works, however I'd say to make it:

```node
if (sres.data.error)
```

Just to be a bit more readable and safer.